packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/telecom-telephony-billing-account-administration.controller.js

```diff
@@ -12,45 +16,52 @@ angular
     ====================================== */
 
       function init() {
-        self.actions = [
-          {
-            name: 'group_billing_options',
-            sref:
-              'telecom.telephony.billingAccount.administration.optionsGroup',
-            text: $translate.instant(
-              'telephony_group_admin_actions_group_billing_options',
-            ),
+        TelephonyMediator.getGroup($stateParams.billingAccount).then(
```

Comment:
```suggestion
        return TelephonyMediator.getGroup($stateParams.billingAccount).then(
```

all `Promise` should be returned so they can be chained :)

Reply:
done too ;)